shared/common-adapters/switch.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import * as React from 'react'
+import * as Styles from '../styles'
+import ClickableBox from './clickable-box'
+import Box, {Box2} from './box'
+import ProgressIndicator from './progress-indicator'
+import Text from './text'
+import SwitchToggle from './switch-toggle'
+
+const Kb = {
+  Box,
+  Box2,
+  ClickableBox,
+  ProgressIndicator,
+  Text,
+}
+
+type Props = {|
+  align?: ?('left' | 'right'), // default to 'left'
+  color?: ?('blue' | 'green'), // default to 'blue'
+  disabled?: ?boolean,
+  label: string,
```

Comment:
would be nice if this were `string | React.Node` for labels like this:
<img width="374" alt="image" src="https://user-images.githubusercontent.com/11968340/55631517-3badf080-5786-11e9-97df-1eb57902179f.png">

Reply:
Sounds good; I'm gonna also add a `labelSubtitle?: ?string` so it'll cover common cases.